ignite/contrib/metrics/test_MARE.py

```diff
@@ -0,0 +1,65 @@
+from ignite.contrib.metrics import MeanAbsoluteRelativeError
```

Comment:
Test file need to be with all other test files :)

Reply:
Fixed.